packages/e2e-test-utils-playwright/src/editor/drag-and-resize.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import type { Locator } from '@playwright/test';
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+type Delta = {
+	x: number;
+	y: number;
+};
+
+/**
+ * Clicks an element, drags a particular distance and releases the mouse button.
+ *
+ * @param {Editor} this
+ * @param {Object} element The puppeteer element handle.
+ * @param {Object} delta   Object containing movement distances.
+ * @param {number} delta.x Horizontal distance to drag.
+ * @param {number} delta.y Vertical distance to drag.
+ *
+ * @return {Promise} Promise resolving when drag completes.
+ */
+export async function dragAndResize(
```

Comment:
Since this util is only used in one test, I don't think we need to include it in this package. Instead, I'd recommend to inline it directly in the test.